atomic_reactor/util.py

```diff
@@ -75,7 +77,9 @@ def to_str(self, registry=True, tag=True, explicit_tag=False,
 
         result = self.repo
 
-        if tag and self.tag:
+        if tag and self.tag and ':' in self.tag:
```

Comment:
`tag.startswith('sha256:')`? That might limit us to one hashing method, but would be a bit more explicit